src/content/1.7/webservice/resources/addresses.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Addresses
+---
+
+# Resources for Addresses
```

Comment:
Would it be possible to add an example of the listing for each resource as well? Only the specified resource is shown for now And I believe some listings have special display (like associations and stuffs) that are worth documenting

Reply:
What do you mean? Something like that 
![image](https://user-images.githubusercontent.com/1462701/93089282-5edb1000-f69b-11ea-9006-872d82a2e65b.png) ?

Reply:
Yep unless they all look very basic and you think they are not very useful?

Reply:
I think it's not really useful :thinking: